packages/babel-parser/ast/spec.md

```diff
@@ -251,12 +251,10 @@ interface NumericLiteral <: Literal {
 ```js
 interface BigIntLiteral <: Literal {
   type: "BigIntLiteral";
-  value: string;
+  value: bigint;
```

Comment:
Technically it should be `string | bigint` now, but maybe nobody noticed and it's fine to keep it the way it is.